src/gpl/src/nesterovBase.cpp

```diff
@@ -2829,15 +2868,17 @@ bool NesterovBase::revertToSnapshot()
 
 void NesterovBaseCommon::moveGCell(odb::dbInst* db_inst)
 {
-  GCell* gcell = getGCellByIndex(db_inst_map_.find(db_inst)->second);
+  GCell* gcell
+      = getGCellByIndex(db_inst_to_nbc_index_map_.find(db_inst)->second);
```

Comment:
I believe you should check for the return value of `find` to ensure the key exists in the map. If it's guaranteed to exists, you could use the `[]` operator.